internal/pkg/cli/pipeline_init.go

```diff
@@ -166,26 +170,35 @@ func (o *initPipelineOpts) Validate() error {
 
```

Comment:
🙏🏼  Would you mind moving `validateInputApp` from `Validate()` to `Ask()` please? 🙏🏼 
Seeing clearly, with some distance from #3322, that `appName` is not optional!